### PR TITLE
change app title and add readme

### PR DESCRIPTION
**Pull request by im-chang**
Changed app.js location and title, added README

### CODE REVIEW

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Please update your prettier config and add single-quotes, semicolons and trailing commas.

**Review by 10thfloor — APPROVED** [review]

File: js/App.js [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow
- */
-
-import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
+import React, { Component } from 'react';
+import { Platform, StyleSheet, Text, View } from 'react-native';
```
Comment (siddharthparmar7) [inline js/App.js:2]: Please remove Platform module import

File: js/App.js [hunk 2]
```diff
@@ -1,13 +1,5 @@
...
 
 const instructions = Platform.select({
```
Comment (siddharthparmar7) [inline js/App.js:4]: Please remove instructions as you are not using it.

File: js/App.js [hunk 3]
```diff
@@ -16,14 +8,11 @@ const instructions = Platform.select({
...
-type Props = {};
-export default class App extends Component<Props> {
+export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text style={styles.welcome}>Welcome to React Native!</Text>
-        <Text style={styles.instructions}>To get started, edit App.js</Text>
-        <Text style={styles.instructions}>{instructions}</Text>
+        <h1>Welcome to AssistList!</h1>
```
Comment (siddharthparmar7) [inline js/App.js:15]: Are you sure, you want this as a pull request? let me know.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Pull request closed without merging**